src/test_driver/tizen/assets.gni

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+import("${build_root}/config/compiler/compiler.gni")
+import("${chip_root}/src/ble/ble.gni")
+
+# Declare arguments used in the build_example.py script (but not declared by
+# any of the imported files above), so we can reuse them for generaing default
+# values for the tests assets.
+declare_args() {
+  chip_enable_wifi = true
+  is_ubsan = false
+}
+
+# Create variant suffix string, as used by the build_examples.py script, based
+# on the build configuration.
+declare_args() {
+  variant = ""
```

Comment:
this internal to build_examples. I don't think this should be here. You can add some defaults, however variant as an argument seems odd. This couples build_examples with build.gn too much.

Generally build_examples.py calls gn, so there is a dependency there, however the reverse should not be done: gn files should not depend on build_examples.py logic.

Reply:
File removed.